examples/grafana-integration/README.md

```diff
@@ -0,0 +1,26 @@
+# Hot R.O.D. - Rides on Demand  - Grafana integration
+
+This is the Hot R.O.D. demo application that consists of the same components as the `examples/hotrod/`, only Grafana and Loki integration is added to this setup, so you can correlate logs and traces in one application.
+
+## Running
+
+### Run everything via `docker-compose`
```

Comment:
This sub-section doesn't add anything, I would merge it into previous one as `## Running via docker-compose`